packages/drivers/routerlicious-driver/src/documentDeltaConnection.ts

```diff
@@ -21,8 +21,8 @@ export enum R11sErrorType {
 function createNetworkError(
     errorMessage: string,
     canRetry: boolean,
-    statusCode?: number,
-    retryAfterSeconds?: number,
+    statusCode: number | undefined,
+    retryAfterSeconds: number | undefined,
```

Comment:
Does anyone pass undefined now? Maybe just make it fully required as `number`?

Reply:
And below in network.ts

Reply:
can't change GenericeNetworkError due to createGenericNetworkError usages